src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -486,12 +462,7 @@ export const SavedSearchAlertEditFormQueryRenderer: React.FC<SavedSearchAlertEdi
           return null
         }
 
-        if (
-          shouldFetchLabelsFromMetaphysics &&
-          props?.artistsConnection &&
-          props.me &&
-          props.viewer
-        ) {
+        if (props?.artistsConnection && props.me && props.viewer) {
```

Comment:
If we will parse pill labels from aggregations, then we have to delete [this](https://github.com/artsy/force/blob/e7abbce7896ef50fa354ea3a939a36d89219af4d/src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx#L465-L477)